package.json

```diff
@@ -1,66 +1,23 @@
 {
   "name": "tala-speech",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "license": "GPL-3.0",
-  "homepage": "http://example.com",
-  "private": true,
-  "dependencies": {
-    "cors": "^2.8.5",
-    "express": "^4.18.1"
-  },
   "scripts": {
-    "start": "parcel src/index.tsx",
-    "build": "parcel build src/index.tsx --no-source-maps --dist-dir dist/",
-    "serve": "npx serve dist/ -l 80",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "predeploy": "npm run build",
-    "deploy": "gh-pages -d build"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
   },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "author": "Talkamatic AB",
+  "packageManager": "yarn@4.0.2",
+  "dependencies": {
+    "speechstate": "https://git@github.com/vladmaraev/speechstate.git",
```

Comment:
I am still not sure that this is a viable practice, given that the dependency will be locked (`yarn.lock`). Also, potentially, there can be newer releases of speechstate that might break compatibility with tala-speech. I would prefer having a specific version of speechstate here.

Reply:
Then we need to solve this in another way. I'm very unskilled in the craft of releases and distribution of JS packages, but let's select the simplest solution and do it.

Reply:
I changed the dependency to 2.0.0. Should work now.

Reply:
BTW, Vlad - does the yarn.lock file need to be in the repo, since it's a generated file?

Reply:
Great, you can see that it builds now!